### PR TITLE
voter count tool blog post

### PR DESCRIPTION
**Pull request by eneemann**
Draft for when the voter count tool blog post goes live

### CODE REVIEW

File: _posts/2021-11-22-voter-counts-blog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The ArcGIS Pro Toolbox can be [downloaded](https://drive.google.com/file/d/1i9OZdYVBkDs-EeFu9JItVTk8rNCs1K0M/view?usp=sharing) by anyone and a [user guide](https://docs.google.com/document/d/1BG-FPROoZkfsoAScDLLSvHGMLuFucdlzaJRILp33rSk) has been created with information on installing and using the tool.  Additionally, we recorded a [UGRC Voting Tools](https://drive.google.com/file/d/1i6ZIwuJjlA0MXdaIxbQcsvUO8nTrkFqe/view) Webinar earlier this year that demonstrates the voter counts tool and data validation checks done by UGRC.
+
+[![Voter Counts Tool]({% link images/voter_tool.PNG %} "Voter Counts Tool")]({% link images/voter_tool.PNG %}){: .flex .flex--center }
+
+If you're interested in using the voter counts tool to help with redrawing your precincts, please reach out to us and we can help set you up.  Questions, issues, or access requests can be directed to Erik Neemann at ([eneemann@utah.gov](mailto:eneemann@utah.gov)).
```
Comment (steveoh) [inline _posts/2021-11-22-voter-counts-blog.md:25]: **nitpick**: please remove your email address and use the data file. More information is available in the readme. https://github.com/agrc/gis.utah.gov#contact-information

**Comment by steveoh** [comment]
Isn't there a relative file checkbox so this isn't necessary? @stdavis can you confirm?

image

File: _posts/2021-11-22-voter-counts-blog.md (hunk 1 above)
Comment (eneemann) [reply _posts/2021-11-22-voter-counts-blog.md:25]: Yeah...for this blog I wasn't sure how to reference my specific contact info via the YML.  Should I reference the zip_codes or something? 
image

Comment (steveoh) [reply _posts/2021-11-22-voter-counts-blog.md:25]: I would add a new one for the voter counts tool

**Comment by eneemann** [comment]
@steveoh I don't see this post showing up in the preview yet.  Did I screw something else up or is it related to the date/time (that I estimated) in the post's header info?

File: _posts/2021-11-22-voter-counts-blog.md (hunk 1 above)
Comment (eneemann) [reply _posts/2021-11-22-voter-counts-blog.md:25]: Okay, I'll give that a shot

**Comment by steveoh** [comment]
> @steveoh I don't see this post showing up in the preview yet. Did I screw something else up or is it related to the date/time (that I estimated) in the post's header info?

I wonder if it's because the date is in the future...

**Comment by eneemann** [comment]
> o

That's what I wondered, so I just bumped up the time to 9:30...15 minutes ago.

**Comment by eneemann** [comment]
> Isn't there a relative file checkbox so this isn't necessary? @stdavis can you confirm?
> 
> image

I'd probably still lean toward leaving this as-is because about 10 people have already downloaded the tool and been granted access to the associated AGOL group.

**Comment by stdavis** [comment]
> Isn't there a relative file checkbox so this isn't necessary? @stdavis can you confirm?
> 
> image

I don't think that we have this restriction with the geocoding toolbox. Perhaps because that is a python toolbox? [This doc](https://pro.arcgis.com/en/pro-app/latest/arcpy/geoprocessing_and_python/adding-a-script-tool.htm) makes it seem like storing a relative path to the script is an option. Maybe there's more to the path issues than just the toolbox reference to the python script?

**Comment by eneemann** [comment]
Looks like the data is available everywhere now (Internal, opensgid, AGOL, OpenData)...shall we merge this?

**Review by steveoh — APPROVED** [review]

**Pull request merged**